gnocchi/tests/test_measures_grouper.py

```diff
@@ -0,0 +1,398 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
```

Comment:
I can't see that this tests is ever executed in CI testing?

Reply:
I checked, indeed, they were not executed, I fixed it as well.

Thanks again.